README.md

```diff
@@ -16,25 +16,39 @@ Or
 
 
 ## Run with Docker
-
+- If you are on your local machine, we recommend that you run this container using Docker.  
+- We have tested running this container using Docker on local mac machines and also on a Pawsey Nimbus instance.
+  
+### On a local (mac) machine
 To run this container:
-
 - Start docker
```

Comment:
How do I do that?

Reply:
Start docker by `double-clicking` on the `Docker` app icon in the `Applications` menu.